src/org/usfirst/frc/team2175/subsystem/DrivetrainSubsystem.java

```diff
@@ -76,16 +75,16 @@ public void arcadeDrive(final double moveValue, final double rotateValue) {
         robotDrive.arcadeDrive(-moveValue, rotateValue);
     }
 
-    private void setGear(final DoubleSolenoid.Value value) {
-        driveShifters.set(value);
+    private void setGear(final boolean forward) {
```

Comment:
@4thwind Let's give this parameter a more descriptive name. Maybe `highGear` instead of `forward`?